gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc

```diff
@@ -1,5 +1,68 @@
-= Publishing Custom Policies to Exchange
+= Upload Custom Policies to Exchange
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+During the custom policy development cycle, you can upload your policy to Exchange to distribute it and manage its version. For more information about Exchange, see xref:exchange::index.adoc[].
+
+When uploading your policy to Exchange you can choose to do either of the following:
+
+* <<publish, Publish a Custom Policy>> +
+Publish a policy if the policy version is still in development. This enables you to test and edit the policy version.
+* <<release, Release a Custom Policy>> +
+Release the policy to create a definitive version of the policy. After releasing the policy, you must create a new policy version to edit the policy.
+
+After you upload your policy to Exchange, use the following tutorials to apply your policy to a Flex Gateway instance:
+
+* xref:policies-pdk-local-deploy.adoc[]
+* xref:policies-pdk-conn-deploy.adoc[]
+
+[[publish]]
+== Publish a Custom Policy
+
+To publish a custom policy, run the publish command from the policy's root folder:
+
+[source,ssh]
+----
+make publish
+----
+
+After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appears similar to the following:
+
+* *groupID*: `<organizationId>`
+* *assetID*: `my-custom-policy-dev` +
```

Comment:
Global - put the + on a line by itself (it affects the line spacing). 
https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-ListContinuation(akaIndentedParagraphs)